DependencyInjection/Configuration.php

```diff
@@ -116,6 +117,7 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->end()
                 ->end()
                 ->arrayNode('api_platform')
+                    ->{class_exists(ApiPlatformBundle::class) ? 'canBeDisabled' : 'canBeEnabled'}()
```

Comment:
Isn't the BC break still here? IMO it should be:
```suggestion
                    ->canBeEnabled()
```

Reply:
It shouldn't, in this way we don't need to put it to true in the recipe and won't throw the exception if APIP is not installed (except if defined by the user in the recipe)

Reply:
But it doesn't solve https://github.com/lexik/LexikJWTAuthenticationBundle/issues/1118: this person is using API Platform and has upgraded to LexixJWT 2.18. Their BC break is that an OpenAPI route has been added just by upgrading.

Reply:
Agree, defaulting to `false` makes sense to me. A new version of the recipe enabling it feels less aggressive as it will only be installed for new projects and recipe upgrades.